usagov/vote-gov-drupal
# Feature/vote 422/develop state page

## Jira ticket

[VOTE-422](https://cm-jira.usa.gov/browse/VOTE-422)

## Description

Develop the state pages according to [the design](https://www.figma.com/file/E70piAGXmi4UzrHs0A90cm/Vote.gov-Redesign-2024?node-id=63%3A2&mode=dev). This does not include global styles such as text coloring. 

## Deployment and testing

### Pre-deploy

1. lando retune
2. cd into the votegov theme and run: npm run build

### QA/Test

1. View each of these state pages and confirm they match the design (except global styling).
http://vote-gov.lndo.site/register/co (online)
http://vote-gov.lndo.site/register/nh (mail)
http://vote-gov.lndo.site/register/as (in person)
http://vote-gov.lndo.site/register/nd (no registration)

## Checklist for the Developer




## Checklist for the Peer Reviewers
